### PR TITLE
PXP-671: [CLI] Implement the CloudSQL integration to the deployment status command

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-671](https://mindvalley.atlassian.net/browse/PXP-671)

### Added
- [x] Display CloudSQL metrics in `wukong deployment status`

### Changed
- [x] Display a message to inform user when they are configuring prod namespace during `wukong application init` 
- [x] Display all the suggested command in the error suggestion with color 
Screenshot 2024-03-12 at 12 36 30 PM

[PXP-671]: https://mindvalley.atlassian.net/browse/PXP-671?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/206#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cli/src/error.rs
Comment (Fadhil) [inline cli/src/error.rs:210]:
```suggestion
                    format!("Check if your `config.toml` file is in valid TOML format.\nThis usually happens when the config file has accidentally been modified or there is a breaking change to the cli config in the new version.\nYou may want to run {} to re-initialise configuration again.", "wukong init".yellow())
```

File: cli/src/error.rs
Comment (Fadhil) [inline cli/src/error.rs:223]:
```suggestion
                    format!("Check if the `.wukong.toml` file is in valid TOML format.\nThis usually happens when the config file has accidentally been modified or there is a breaking change to the application config in the new version.\nYou may want to run {} to re-initialise configuration again.", "wukong application init".yellow())
```

File: cli/src/commands/deployment/status.rs
Comment (Fadhil) [inline cli/src/commands/deployment/status.rs:218]:
```suggestion
                    "* Appsignal status is not displayed because {}",
```

File: cli/src/commands/deployment/status.rs
Comment (Fadhil) [inline cli/src/commands/deployment/status.rs:236]:
```suggestion
                colored_println!("* CloudSQL status is not displayed because {}", reason.bold());
```

**Review by Fadhil — CHANGES_REQUESTED** [review]
Minor grammar changes

**Review by Fadhil — DISMISSED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/206#pullrequestreview-1933530709" title="Approved">Fadhil</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/206#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cli/src/commands/application/init.rs
Comment (mfauzaan) [inline cli/src/commands/application/init.rs:53]: why not use this way to reduce the number of lines

```suggestion
    eprintln!("\nNext is to configure the prod namespace for your application.\n");
```

**Review by mfauzaan — CHANGES_REQUESTED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/206#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/206#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:54]:
```suggestion
```

File: cli/src/commands/application/init.rs
Comment (onimsha) [inline cli/src/commands/application/init.rs:52]:
```suggestion
```

**Review by mfauzaan — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/206#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/206#pullrequestreview-1935576239" title="Approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
